bin/afl_wrapper.sh

```diff
@@ -78,12 +78,12 @@ function afl_env() {
                 use_sigaltstack=1"
 
     # Keep them sorted
+    # AFL_DRIVER_DONT_DEFER=1 \ Reuse if there is an issue with deferred mode
```

Comment:
Why not just remove the flag? There's always the git history to refer to how things were done in the past.
```suggestion
```

Reply:
Purely a visual feedback for me but I agree removing is way more cleaner